unstructured/ingest/connector/vectara.py

```diff
@@ -0,0 +1,246 @@
+import datetime
+import json
+import traceback
+import typing as t
+import uuid
+from dataclasses import dataclass
+
+import requests
+
+from unstructured.ingest.enhanced_dataclass import enhanced_field
+from unstructured.ingest.error import DestinationConnectionError
+from unstructured.ingest.interfaces import (
+    AccessConfig,
+    BaseConnectorConfig,
+    BaseDestinationConnector,
+    BaseIngestDoc,
+    WriteConfig,
+)
+from unstructured.ingest.logger import logger
+from unstructured.staging.base import flatten_dict
+
+
+@dataclass
+class VectaraAccessConfig(AccessConfig):
+    oauth_client_id: str = enhanced_field(sensitive=True)
+    oauth_secret: str = enhanced_field(sensitive=True)
+
+
+@dataclass
+class SimpleVectaraConfig(BaseConnectorConfig):
+    access_config: VectaraAccessConfig
+    customer_id: t.AnyStr = None
+    corpus_name: t.AnyStr = "vectara-unstructured"
+    corpus_id: t.AnyStr = None
+
+
+@dataclass
+class VectaraWriteConfig(WriteConfig):
+    batch_size: int = 100
```

Comment:
Didn't see this exposed in the CLI options.

Reply:
Ha. Yes. Leftover from elasticsearch. I'm going to ditch it.